components/molecules/MDRenderer.js

```diff
@@ -0,0 +1,11 @@
+import { ReactMarkdown } from "react-markdown/lib/react-markdown";
+import styles from "../../styles/MDRenderer.module.css";
+
+export default function MarkdownLayout(props) {
+    return (
+        // this will get expanded on when styling for customm renders are included
+        <ReactMarkdown className={styles.markdown}>
+            {props.markdown}
+        </ReactMarkdown>
```

Comment:
MDRenderer should also be an organism as it will be taking in molecules

Reply:
That's exactly what the current code is doing, passing it as the children prop.
As it's only using one other component, molecules is a perfectly fine place for it - and it ultimately as no impact on the code.

Reply:
react-markdown docs always show passing it inline with `children={*insert markdown prop here*}` so was going by that.

As discussed in our meeting, it will be importing multiple molecules to support the upcoming code/design views so it should be an organism. Moving it to molecules had no effect on the code to begin with.

Reply:
The docs do reference it as I've done it.
And as you say, it has no effect on the code, so it can be moved at such a time that it's logical for it to move.